specification/monitor/resource-manager/Microsoft.Insights/preview/2021-05-01-preview/examples/getDiagnosticSetting.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "resourceUri": "subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6",
+    "name": "mysetting",
+    "api-version": "2021-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6/providers/microsoft.insights/diagnosticSettings/service",
```

Comment:
>service [](http://example.com/codeflow?start=184&length=7)

The resource name should match the one in the request. 'mysetting'.

Reply:
fixed